OMGVA PoS/Data layer/DTOs/Tax/UpdateTaxRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OMGVA_PoS.Data_layer.DTOs
+{
+    public class UpdateTaxRequest
```

Comment:
Also very close to tax model.

Reply:
Although it's close, it's not the same. `Tax` model is our internal representation, it should not be exposed. Also, since this was for partial updates, all of the fields are nullable. Both `Tax` model and `CreateTaxRequest` have required fields

Reply:
Fine then.